<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General Styles */
QWidget {
    background-color: #fbfbfe;
    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
    font-size: 18px;
    color: #050315;
}

/* QLabel Styles */
QLabel {
    color: #050315;
    font-size: 18px;
    font-weight: bold;
}

/* QLineEdit Styles */
QLineEdit {
    background-color: #dedcff;
    border: 1px solid #2118d8;
    border-radius: 5px;
    padding: 5px;
    font-size: 18px;
    color: #050315;
}

/* QTextEdit Styles */
QTextEdit {
    background-color: #dedcff;
    border: 4px solid #2118d8;
    border-radius: 5px;
    padding: 5px;
    font-size: 18px;
    color: #050315;
}

/* QTableWidget Styles */
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #2118d8;
    border-radius: 5px;
    padding: 5px;
    font-size: 18px;
    color: #050315;
    gridline-color: #2118d8;
}

QHeaderView::section {
    background-color: #0094ff;
    color: #fbfbfe;
    padding: 5px;
    border: none;
}

/* QPushButton Styles */
QPushButton {
    background-color: #0094ff;
    color: #fbfbfe;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 18px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #433bff;
}

QPushButton:pressed {
    background-color: #2118d8;
    padding-left: 15px;
    padding-top: 15px;
}

/* Specific Button Styles */
QPushButton#addAppointmentButton,
QPushButton#editAppointmentButton,
QPushButton#deleteAppointmentButton,
QPushButton#addRecordButton,
QPushButton#editRecordButton,
QPushButton#deleteRecordButton,
QPushButton#addSlotButton,
QPushButton#editSlotButton,
QPushButton#deleteSlotButton {
    background-color: #0094ff;
    color: #fbfbfe;
}

QPushButton#addAppointmentButton:hover,
QPushButton#editAppointmentButton:hover,
QPushButton#deleteAppointmentButton:hover,
QPushButton#addRecordButton:hover,
QPushButton#editRecordButton:hover,
QPushButton#deleteRecordButton:hover,
QPushButton#addSlotButton:hover,
QPushButton#editSlotButton:hover,
QPushButton#deleteSlotButton:hover {
    background-color: #433bff;
}

QPushButton#addAppointmentButton:pressed,
QPushButton#editAppointmentButton:pressed,
QPushButton#deleteAppointmentButton:pressed,
QPushButton#addRecordButton:pressed,
QPushButton#editRecordButton:pressed,
QPushButton#deleteRecordButton:pressed,
QPushButton#addSlotButton:pressed,
QPushButton#editSlotButton:pressed,
QPushButton#deleteSlotButton:pressed {
    background-color: #2118d8;
    padding-left: 15px;
    padding-top: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>username</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>age</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUpdate">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDelete">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelUsername">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditUsername"/>
         </item>
         <item>
          <widget class="QLabel" name="labelAge">
           <property name="text">
            <string>Age:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAge"/>
         </item>
         <item>
          <widget class="QLabel" name="labelType">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxType">
           <item>
            <property name="text">
             <string>Doctor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Patient</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nurse</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Appointments</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>981</width>
         <height>611</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Slot Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Patient Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Contact number</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Doctor</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButtonDeleteAppointment">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>630</y>
         <width>215</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
